mozilla-mobile/android-components/l10n.toml

```diff
@@ -0,0 +1,73 @@
+
+basepath = "."
+
+locales = [
+    "de",
+    "es",
+    "fr",
+    "it",
+    "ja",
+    "ko",
+    "zh-CN",
+    "zh-TW",
+]
+
+# Expose the following branches to localization
+# Changes to this list should be announced to the l10n team ahead of time.
+branches = [
+    "master",
+]
+
+[env]
+
+[[paths]]
+  reference = "components/browser/engine-system/src/main/res/values/strings.xml"
+  l10n = "components/browser/engine-system/src/main/res/values-{android_locale}/strings.xml"
```

Comment:
Are we mapping Mozilla locale to Android locales?

Reply:
Yes, that's part of project config. In pontoon, we use our locale codes, and they're mapped back and forth during serialization.